.gitignore

```diff
@@ -1 +1,2 @@
 *.gem
+Gemfile.lock
```

Comment:
https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/